### DISCUSSION TITLE
Group Assignment #1

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#7] [Assignments]
## Due 4/6 at 2pm PST
Post the link to your group repository alongside group members here to complete [group assignment #1](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week1/group_assignment/)

Ex. Albert, Yoh: https://github.com/albertkun/my-example-project/

**Comment by ashley-lu** [comment]
Ash, Haven, Shubham: https://github.com/shubhgupta2018/uclgay

**Comment by daikis12** [comment]
Angeline, Daiki, & Lindsey: https://github.com/daikis12/group-8/blob/main/README.md

**Comment by penelopelamb** [comment]
Penelope, Sine, Alexis: https://github.com/spolcharoen/aa191a-group-envi

**Comment by ananyakodali17** [comment]
Jenny, Ananya: https://github.com/ananyakodali17/tax-redistribution-asian-am-191a

**Comment by stefuntran** [comment]
Diana,Steven
https://stefuntran.github.io/AAPI-Discrimination/

**Comment by XiongHer** [comment]
Ngoc, Willa, Xiong - link below

**Reply by ntran27** [reply]
Use this link instead: https://ntran27.github.io/22s-asiaam-191a-group/week1-group-lab

**Comment by bittertang** [comment]
Nishaanth, Ivy: https://github.com/bittertang/cuddly-broccoli/blob/main/readme.md

**Comment by kaittli** [comment]
Antony, Kaitlyn: https://github.com/kaittli/aanhpi-healthcare/blob/main/README.md

**Comment by kericsp** [comment]
Kristen, Emma, Keri: https://github.com/emmashanson/asianam191Akristenemmakeri/blob/main/README.md
